src/unstract/sdk/file_storage/env_helper.py

```diff
@@ -0,0 +1,46 @@
+import json
+import logging
+import os
+
+from unstract.sdk.exceptions import FileStorageError
+from unstract.sdk.file_storage.constants import CredentialKeyword, StorageType
+from unstract.sdk.file_storage.impl import FileStorage
+from unstract.sdk.file_storage.permanent import PermanentFileStorage
+from unstract.sdk.file_storage.provider import FileStorageProvider
+from unstract.sdk.file_storage.shared_temporary import SharedTemporaryFileStorage
+
+logger = logging.getLogger(__name__)
+
+
+class EnvHelper:
+    @staticmethod
+    def get_storage(storage_type: StorageType, env_name: str) -> FileStorage:
+        try:
+            file_storage_creds = json.loads(os.environ.get(env_name))
+            provider = FileStorageProvider(
+                file_storage_creds[CredentialKeyword.PROVIDER]
+            )
+            if CredentialKeyword.CREDENTIALS in file_storage_creds:
+                credentials = file_storage_creds[CredentialKeyword.CREDENTIALS]
+            else:
+                credentials = {}
```

Comment:
@gaya3-zipstack The else condition can be avoided if we initialize `credentials` at the beginning of the method. This way looks more optimal.

Reply:
Done. Thank you